parlai/mturk/core/react_server/dev/components/core_components.jsx

```diff
@@ -35,6 +35,16 @@ class ChatMessage extends React.Component {
       float_loc = 'right';
       alert_class = 'alert-info';
     }
+    let duration = null;
+    if (this.props.duration !== undefined) {
+      let duration_seconds = Math.floor(this.props.duration / 1000) % 60;
+      let duration_minutes = Math.floor(this.props.duration / 60000);
+      let min_text = duration_minutes > 0 ? duration_minutes + ' min' : '';
+      let sec_text = duration_seconds > 0 ? duration_seconds + ' sec' : '';
+      duration = <small>
+        <br /><i>Duration: </i>{min_text + ' ' + sec_text}
+      </small>;
+    }
```

Comment:
so this shows the amount of time a turker took to write a message?

Reply:
Yup, but only in review mode. Turkers can't see this